seacrowd/sea_datasets/oscar_2201/oscar_2201.py

```diff
@@ -0,0 +1,350 @@
+}
+
+@inproceedings{OrtizSuarezSagotRomary2019,
+    author    = {Pedro Javier {Ortiz Su{'a}rez} and
+                Benoit Sagot and
+                Laurent Romary},
+    title     = {Asynchronous pipelines for processing huge corpora on medium to low resource infrastructures},
+    series    = {Proceedings of the Workshop on Challenges in the Management of Large Corpora
+                (CMLC-7) 2019. Cardiff, 22nd July 2019},
+    editor    = {Piotr Bański and
+                Adrien Barbaresi and
+                Hanno Biber and
+                Evelyn Breiteneder and
+                Simon Clematide and
+                Marc Kupietz and
+                Harald L{"u}ngen and
+                Caroline Iliadi},
+    publisher = {Leibniz-Institut f{"u}r Deutsche Sprache},
+    address   = {Mannheim},
+    doi       = {10.14618/ids-pub-9021},
+    url       = {http://nbn-resolving.de/urn:nbn:de:bsz:mh39-90215},
+    pages     = {9 -- 16},
+    year      = {2019},
+    abstract  = {Common Crawl is a considerably large, heterogeneous multilingual corpus
+    comprised of crawled documents from the internet, surpassing 20TB of data and
+    distributed as a set of more than 50 thousand plain text files where each contains
+    many documents written in a wide variety of languages. Even though each document has a
+    metadata block associated to it, this data lacks any information about the language in
+    which each document is written, making it extremely difficult to use Common Crawl for
+    monolingual applications. We propose a general, highly parallel, multithreaded
+    pipeline to clean and classify Common Crawl by language; we specifically design it so
+    that it runs efficiently on medium to low resource infrastructures where I/O speeds
+    are the main constraint. We develop the pipeline so that it can be easily reapplied to
+    any kind of heterogeneous corpus and so that it can be parameterised to a wide range
+    of infrastructures. We also distribute a 6.3TB version of Common Crawl, filtered,
+    classified by language, shuffled at line level in order to avoid copyright issues, and
+    ready to be used for NLP applications.},
+    language  = {en}
+}
+"""
+
+_DATASETNAME = "oscar_2201"
+_DESCRIPTION = """\
+OSCAR or Open Super-large Crawled Aggregated coRpus is a huge multilingual corpus
+obtained by language classification and filtering of the Common Crawl corpus using
+the ungoliant architecture. Data is distributed by language in both original and
+deduplicated form.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/oscar-corpus/OSCAR-2201"
+_LICENSE = f"""{Licenses.OTHERS.value} | \
+These data are released under this licensing scheme \
+We do not own any of the text from which these data has been extracted. \
+We license the actual packaging of these data under the Creative Commons CC0 license ("no rights reserved") http://creativecommons.org/publicdomain/zero/1.0/ \
```

Comment:
Isn't it a `CC0 1.0` License?

Reply:
Hmm, I believe `CC0 1.0` is the packaging and not the data itself. Not sure though.

Reply:
Here's my understanding to the licensing of this data, according to my observation:
1. HF License tag uses `CC0 1.0`
2. The license Section uses `CC0 1.0` as well (the text that you copied into this dataloader)
3. The only mentions of other `TERMS OF USE` based on their HF repo is on one of their derivative work from OSCAR (under this `AbadjiOrtizSuarezRomaryetal.2021` citations). This is a bit unclear whether the portion of this work is being added into the HF dataset, however their Abstraction mentions their data (which is a research-only corpus) is different than OSCAR, hence making the initial LICENSE of `CC0 1.0` still holds

cc @SamuelCahyawijaya @holylovenia , do you have any comments/POV?

Reply:
Since this data itself derived from CC data, we have another alternative to follow the LICENSE of CC data. But anw, I saw other derivatives of CC data have their own licensing

Reply:
@akhdanfadh @sabilmakbar : Oh, I think in this case we can use CC0 for OSCAR. 
It is also explicitly mentioned in various sources:
1. https://oscar-project.org/
2. https://paperswithcode.com/dataset/oscar
3. https://paperswithcode.com/dataset/oscar.